lib/Timer/index.js

```diff
@@ -29,6 +29,12 @@ class Timer {
     this.intervalId = undefined
   }
 
+  resetTimer () {
+    clearInterval(this.intervalId)
+    this.intervalId = undefined
+    this.startTime = undefined
```

Comment:
Shouldn't it also clear `this.endTime`?

Reply:
Haha yes, I don't know why I thought that you would never try to end the timer before resetting it.